### PR TITLE
fix: typos and codeblocks

### PR DESCRIPTION
**Pull request by brnhensley**
there were a lot of cut/paste errors in here, like stray characters that make no sense or commented out code samples

I added `[output]` to some code blocks so it formats as terminal output, those strings won't be rendered literally. It looks like people were commenting out the code you should run to highlight the output, but that's something you can do with the codeblock with `[output]`.

I also lined up some code comments because I think it's easier to read

Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

**Review by ally-sassman — APPROVED** [review]

**Review by ally-sassman — DISMISSED** [review]

File: src/content/docs/more-integrations/terraform/terraform-modules.mdx [hunk 1]
```diff
@@ -106,45 +105,44 @@ Try testing your configurations, run `terraform plan` and
...
 
-
 <Step>
 
 ## Create a Variable
 
 Variables pass details into your module and set default values.
 
-First, at the top of your HostConditions provider.tf create a new variable:
+First, at the top of your `HostConditions` `provider.tf` create a new variable:
```
[the remark this answers is not in the document]
Comment (brnhensley) [reply src/content/docs/more-integrations/terraform/terraform-modules.mdx:140]: yes,  that's a file in the folder, I'll add the `/`

[the remark this answers is not in the document]
Comment (brnhensley) [reply src/content/docs/more-integrations/terraform/terraform-modules.mdx]: fixed, I wasn't positive that `HostConditions/provider.tf` would always be correct, but from the context it looks like it. I was playing it safe originally and thinking of it like `HostConditions` possessive of `provider.tf` in some way. My grammer is bad though, I guess that would have needed an `'` somehow..

**Review by ally-sassman — APPROVED** [review]

**Comment by brnhensley** [comment]
guess I broke it 🤷

**Comment by brnhensley** [comment]
gotcha, thanks for the update

**Review by akristen — APPROVED** [review]
Got it to deploy here! https://app.netlify.com/sites/docs-website-netlify/deploys/6614224d5dadb6009a42bb8a Merging it in

**Comment by brnhensley** [comment]
Thanks, those terraform codeblocks look nice now:

image

**Pull request merged**